src/quemb/shared/config.py

```diff
@@ -27,12 +28,17 @@
 
 from quemb.shared.helper import add_docstring
 
+logging.basicConfig(
+    level=logging.CRITICAL + 1,  # Disables all logging
+    format="%(asctime)s [%(levelname)s] %(name)s: %(message)s",
+)
+
+
 DEFAULT_RC_PATH: Final = Path("~/.quembrc.yml")
 
 
 @define
 class Settings:
-    PRINT_LEVEL: int = 5
```

Comment:
Quick question about this part (which I think is related to @mcocdawc 's discussion) -- do we still retain ability to specify the default log level? i.e., if the user puts
```
logging.getLogger().setLevel(logging.DEBUG)
```
on top of their script, does it set to debug, or critical? I'm not super familiar with how logging works, so @cariluo your input would be helpful